Source/WebKit/UIProcess/Launcher/glib/BubblewrapLauncher.cpp

```diff
@@ -302,6 +302,28 @@ static void bindSndio(Vector<CString>& args)
     bindIfExists(args, sndioHomeDir.get(), BindFlags::ReadWrite);
 }
 
+static void bindPipewire(Vector<CString>& args)
+{
+    const char* xdgRuntimeDir = g_get_user_runtime_dir();
+    const char* pwRuntimeDir = g_getenv("PIPEWIRE_RUNTIME_DIR");
+    const char* pwRemote = g_getenv("PIPEWIRE_REMOTE");
+
+    if (!pwRemote)
+        pwRemote = "pipewire-0";
+
+    if (pwRuntimeDir) {
+        GUniquePtr<char> pwRuntimeFile(g_build_filename(pwRuntimeDir, pwRemote, nullptr));
+        bindIfExists(args, pwRuntimeFile.get(), BindFlags::ReadWrite);
+    } else {
+        GUniquePtr<char> xdgRuntimeFile(g_build_filename(xdgRuntimeDir, pwRemote, nullptr));
+        bindIfExists(args, xdgRuntimeFile.get(), BindFlags::ReadWrite);
+    }
+
+    // System-wide pipewire
```

Comment:
Interestingly, I notice we don't have a system instance of Pipewire on Fedora. I guess that's an optional configuration that's not really expected to be used on desktops? OK, whatever. If it exists, applications should surely be able to use it.